aws/rust-runtime/aws-types/src/sdk_config.rs

```diff
@@ -461,6 +465,52 @@ impl Builder {
         self
     }
 
+    /// When true, send this request to the FIPS-compliant regional endpoint.
+    ///
+    /// If the configured endpoint does not have a FIPS compliant endpoint, dispatching
+    /// the request will return an error.
+    ///
+    /// **Note**: Not all services and regions support FIPS. If a service does not support FIPS,
+    /// this setting will have no effect.
+    pub fn use_fips(mut self, use_fips: bool) -> Self {
+        self.set_use_fips(Some(use_fips));
+        self
+    }
+
+    /// When true, send this request to the FIPS-compliant regional endpoint.
+    ///
+    /// If the configured endpoint does not have a FIPS compliant endpoint, dispatching
+    /// the request will return an error.
+    ///
+    /// **Note**: Not all services and regions support FIPS. If a service does not support FIPS,
+    /// this setting will have no effect.
+    pub fn set_use_fips(&mut self, use_fips: Option<bool>) -> &mut Self {
+        self.use_fips = use_fips;
+        self
+    }
+
+    /// When true, send this request to the dual-stack endpoint.
+    ///
+    /// If the configured endpoint does not support dual-stack, the request MAY return an error.
+    ///
+    /// **Note**: Not all services and regions support dual-stack. If a service does not support
+    /// dual-stack, this setting will have no effect.
+    pub fn use_dual_stack(mut self, use_dual_stack: bool) -> Self {
+        self.set_use_dual_stack(Some(use_dual_stack));
+        self
+    }
+
+    /// When true, send this request to the dual-stack endpoint.
+    ///
+    /// If the configured endpoint does not support dual-stack, the request MAY return an error.
+    ///
+    /// **Note**: Not all services and regions support dual-stack. If a service does not support
+    /// dual-stack, this setting will have no effect.
+    pub fn set_use_dual_stack(&mut self, use_dual_stack: Option<bool>) -> &mut Self {
+        self.use_dual_stack = use_dual_stack;
+        self
+    }
```

Comment:
I wish we could set the `ConfigLoader` docs to pull from these docs so that we only have to update them in one place. If you end up re-wording the `ConfigLoader` docs, don't forget these ones.

Reply:
done!